doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -144,81 +144,8 @@ kubectl logs raycluster-complete-logs-head-xxxxx -c fluentbit
 [KubDoc]: https://kubernetes.io/docs/concepts/cluster-administration/logging/
 [ConfigLink]: https://raw.githubusercontent.com/ray-project/ray/releases/2.4.0/doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml
 
-## Using structured logging
 
-The metadata of tasks or actors may be obtained by Ray's :ref:`runtime_context APIs <runtime-context-apis>`.
-Runtime context APIs help you to add metadata to your logging messages, making your logs more structured.
-
-```python
-import ray
-# Initiate a driver.
-ray.init()
-
- @ray.remote
-def task():
-    print(f"task_id: {ray.get_runtime_context().task_id}")
-
-ray.get(task.remote())
-```
-
-```bash
-(pid=47411) task_id: TaskID(a67dc375e60ddd1affffffffffffffffffffffff01000000)
-```
 ## Redirecting Ray logs to stderr
+By default, Ray logs are written to files under the ``/tmp/ray/session_*/logs`` directory. It may not be ideal if the log processing tool needs log to be written to stderr in order for them to be captured. View {ref}`configuring logging <redirect-to-stderr>` for details on how to redirect all the logs to stderr of the host pods instead.
```

Comment:
```suggestion
By default, Ray writes logs to files under the ``/tmp/ray/session_*/logs`` directory. Capturing logs from stderr may not be ideal for log processing tools. View {ref}`configuring logging <redirect-to-stderr>` for details on how to redirect logs to the stderr of the host pods instead.
```

Reply:
> Capturing logs from stderr may not be ideal for log processing tools.

RE this, what we want to say here is: `Writing to log files is not ideal if users or the log processing tools prefer to capture logs from stderr.`